config/prow/cluster/400-boskos-janitor.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
     app: boskos-janitor
   namespace: test-pods
 spec:
-  replicas: 35  # Number of distributed janitor instances
+  replicas: 8  # Number of distributed janitor instances; k8s has ~1/16 as many projects
```

Comment:
Shall we be a bit less aggressive? Maybe change to 15 first?

Reply:
We only have one cleaning

Reply:
Add each supports multiple cleaning threads AFAICT